translation-web/src/main/java/eu/europeana/api/translation/web/TranslationSchemaLocation.java

```diff
@@ -0,0 +1,10 @@
+package eu.europeana.api.translation.web;
+
+import eu.europeana.api.commons.JsonSchemaLocation;
+
+public interface TranslationSchemaLocation extends JsonSchemaLocation {
```

Comment:
this class contains our RND server location. These values should be added to the config file and accessed from there. 

You can have the nested properties here. 
But the only problem is if in the future we update or change the name, you need to make code changes and make a new release. If we add all this in the config file, we can easily redeploy. Better to add this in the config file, and access them. 

I think we already have the TranslationConfig class where you can access them and use them in controller